packages/bot-cleaning/src/MarketCleaner.ts

```diff
@@ -353,15 +369,14 @@ export class MarketCleaner {
 
   #createCollectParams(
     offer: Market.Offer,
-    takerWants: Big,
-    takerGives: Big
-  ): Market.SnipeParams["targets"] {
+    takerWants: Big
+  ): Market.CleanParams["targets"] {
     return [
       {
         offerId: offer.id,
-        takerWants: takerWants.toFixed(),
-        takerGives: takerGives.toFixed(),
-        gasLimit: maxGasReq,
+        takerWants: takerWants,
+        tick: offer.tick.toString(),
```

Comment:
Why toString ? probably safe, but we were using toFixed before

Reply:
Now it is a BigNumber so no toFixed as it is a uint